### PR TITLE
Portofolio profile image backend

### PR DESCRIPTION
**Pull request by sadinamantha**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -43,6 +43,38 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <!-- JUnit 5 dependency -->
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.7.0</version>
```
Comment (isobelj715) [inline pom.xml:51]: Why are you adding this and why using an older version

File: src/main/resources/com/example/addressbook/my-portfolios-overview.fxml
Comment (isobelj715) [inline src/main/resources/com/example/addressbook/my-portfolios-overview.fxml:1]: We already have a my-portfolios-view that displays the portfolios. dont really understand where this fits into our project

File: src/test/java/InsidePortfolioController.java
Comment (isobelj715) [inline src/test/java/InsidePortfolioController.java:1]: There is no InsidePortfolioController file, Jimmy changed the name to PortfolioContentController

File: src/main/resources/com/example/addressbook/my-portfolios-overview.fxml
Comment (isobelj715) [inline src/main/resources/com/example/addressbook/my-portfolios-overview.fxml:1]: We already have a my-portfolios-view that displays the portfolios. dont really understand where this fits into our project

File: pom.xml [hunk 2]
```diff
@@ -43,6 +43,38 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+
+        <!-- JUnit 5 dependency -->
```
Comment (isobelj715) [inline pom.xml:47]: Why are you adding this and why using an older version

File: src/main/java/com/example/addressbook/controller/UploadPortfolioController.java [hunk 3]
```diff
@@ -148,11 +158,50 @@ public void onCreatePortfolio(ActionEvent event) {
...
+                portfolioImageView.setImage(image);  // Update the portfolio image preview
+
+            } catch (Exception e) {
+                showAlert("Error", "Selected file is not a valid image.");
+
+
+            }
+        }
+    }
+
```
Comment (isobelj715) [inline src/main/java/com/example/addressbook/controller/UploadPortfolioController.java:201]: added functionality that we never discussed? The page that displays portfolios has already been made as a list format.

**Review by isobelj715 — CHANGES_REQUESTED** [review]
please check this stuff and lmk why you want to make the changes.

File: src/main/resources/com/example/addressbook/my-portfolios-overview.fxml
Comment (sadinamantha) [reply src/main/resources/com/example/addressbook/my-portfolios-overview.fxml:1]: hey so sorry this was done way back and I forgot to delete that one. my bad

File: pom.xml (hunk 2 above)
Comment (sadinamantha) [reply pom.xml:47]: already deleted the repeated dependencies for junit :)

File: src/main/resources/com/example/addressbook/my-portfolios-overview.fxml
Comment (isobelj715) [reply src/main/resources/com/example/addressbook/my-portfolios-overview.fxml:1]: Ahh ok 
Could you pls start a new branch off the current integration and just copy over the unit test and then redo this push and merge. just cos I think leaving in all the stuff you forgot to delete might break some of our other code.

File: pom.xml (hunk 2 above)
Comment (isobelj715) [reply pom.xml:47]: as in you deleted the one you added? the dependency we already have is a more current version so dont think we need to change it

File: src/main/resources/com/example/addressbook/my-portfolios-overview.fxml
Comment (sadinamantha) [reply src/main/resources/com/example/addressbook/my-portfolios-overview.fxml:1]: okkies thanks heaps, I'll create a new branch :)

**Pull request closed without merging**